routes/definitions.js

```diff
@@ -69,15 +69,25 @@ router.post(
 router.post(
   '/',
   asyncMiddleware(async (request, response) => {
-    const coordinatesList = request.body.map(entry => EntityCoordinates.fromString(entry))
-    if (coordinatesList.length > 1000)
-      return response.status(400).send(`Body contains too many coordinates: ${coordinatesList.length}`)
-    // if running on localhost, allow a force arg for testing without webhooks to invalidate the caches
-    const force = request.hostname.includes('localhost') ? request.query.force || false : false
-    const result = await definitionService.getAll(coordinatesList, force)
-    response.status(200).send(result)
+    if (Array.isArray(request.body)) await listDefinitions(request, response)
```

Comment:
why `await` here. can you just return the promise that comes from the *Definitions function?